### PR TITLE
Add crawler for Computer Networks 2019 (AC Pang)

### PR DESCRIPTION
**Pull request by pr3pony**
meow

### CODE REVIEW

**Comment by oToToT** [comment]
maybe you could also choose a color to be used in [index.html](https://github.com/brianbbsu/NTU-Announcement-Crawler/blob/master/templates/index.html)

**Comment by pr3pony** [comment]
how to do that

**Comment by oToToT** [comment]
@prprprpony you could find an example in [TocasUI](https://tocas-ui.com/collections/table/).
and modify it like [this](https://github.com/brianbbsu/NTU-Announcement-Crawler/blob/master/templates/index.html#L43).

File: crawlers/CN2019_AC_Pang.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                classname = "Computer Networks 2019 (AC Pang)"
+                anno = Announcement(title=title, content=content, date=date,
+                                    url=url, pos=it, classname=classname)
+                annos.append(anno)
+                it += 1
+        return annos
+
+
+if __name__ == '__main__':
+    c = CN2019Crawler()
```
Comment (brianbbsu) [inline crawlers/CN2019_AC_Pang.py]: I think this is suppose to be `CN2019_AC_Pang_Crawler()`

Comment (pr3pony) [reply crawlers/CN2019_AC_Pang.py]: fixed

**Comment by brianbbsu** [comment]
> maybe you could also choose a color to be used in [index.html](https://github.com/brianbbsu/NTU-Announcement-Crawler/blob/master/templates/index.html)

@oToToT 

Actually I don't think it's a good idea to hard-code crawler -> color relation. Maybe we should create a list of color, then choose the color to use base on, like, index of the crawler when sorted alphabetically, module length of the color list.

This way, we can have different color for each crawler, also making the color presistent as long as the list of crawlers does not change.

**Comment by brianbbsu** [comment]
I think this is a bug (the announcement should stop at 「...領取加簽授權碼。」). 

I haven't take time investigate it. 

Screenshot from 2019-10-25 02-44-14

**Comment by brianbbsu** [comment]
Looks good. I'll merge it.

**Pull request merged**